src/server.ts

```diff
@@ -1,10 +1,12 @@
+// sort-imports-ignore
```

Comment:
Maybe simply add a comment explaining that in order for dotenv to initialise the variables they need to be imported first before anything else.